modules/accountplan/view/record.inc

```diff
@@ -94,12 +94,65 @@ function remove_hanging_firmaids() {
     $_lib['db']->db_query($query);
 }
 
+// Create NO:ORGNR and NO:VAT if they do not already exist for this accountplan
 function create_new_noorgnr_and_vat($AccountPlanID, $OrgNumber = '') {
     global $_lib;
-    $query = "INSERT INTO accountplanscheme (AccountPlanID, FakturabankSchemeID, SchemeValue, CountryCode) VALUES
-              ('". $AccountPlanID ."', (SELECT FakturabankSchemeID FROM fakturabankscheme WHERE SchemeType = 'NO:ORGNR'), '". $OrgNumber ."', 'NO'),
-              ('". $AccountPlanID ."', (SELECT FakturabankSchemeID FROM fakturabankscheme WHERE SchemeType = 'NO:VAT'), '', 'NO');";
-    $_lib['db']->db_query($query);
+    $select_query = "
+      SELECT
+        *
+      FROM
+        accountplanscheme aps
+        LEFT JOIN
+        fakturabankscheme fbs
+        ON
+          aps.FakturabankSchemeID = fbs.FakturabankSchemeID
+      WHERE
+        fbs.SchemeType = 'NO:ORGNR' AND
```

Comment:
Use the same queries just replace the Firma ID name by looping throught an array with ('NO:ORGNR', 'NO:VAT')